conf/messages.en

```diff
@@ -609,3 +609,32 @@ haveYouRegisteredForVATMOSS.subtext = You’ll have received a VAT MOSS identifi
 haveYouRegisteredForVATMOSS.Yes = Yes - I’ve already registered for VAT MOSS
 haveYouRegisteredForVATMOSS.No = No - I’m not registered for VAT MOSS yet
 haveYouRegisteredForVATMOSS.error.required = Select yes or no
+
+#######################################################
+##  UseEmployersPAYE
+#######################################################
+useEmployersPAYE.title = Use your PAYE for employers account
+useEmployersPAYE.heading = Use your PAYE for employers account
```

Comment:
should employers have an '

Reply:
There are a few in this file, just check them all

Reply:
spike raised